lib/http-bridge/test/integration/Test/Integration/Scenario/Transactions.hs

```diff
@@ -8,31 +11,82 @@ module Test.Integration.Scenario.Transactions
 import Prelude
 
 import Cardano.Wallet.Api.Types
-    ( ApiWallet )
+    ( ApiAddress, ApiTransaction, ApiWallet )
+import Cardano.Wallet.Primitive.Types
+    ( Direction (..), TxStatus (..) )
 import Data.Generics.Internal.VL.Lens
-    ( view )
+    ( view, (^.) )
 import Test.Hspec
     ( SpecWith, it )
 import Test.Integration.Framework.DSL
     ( Context
     , Headers (..)
     , Payload (..)
+    , amount
     , balanceAvailable
     , balanceTotal
+    , direction
+    , expectEventually
     , expectFieldEqual
+    , expectResponseCode
+    , expectSuccess
     , fixtureWallet
+    , json
     , request
+    , status
+    , unsafeRequest
     , verify
     , walletId
     )
 
+import qualified Network.HTTP.Types.Status as HTTP
+
 spec :: SpecWith Context
 spec = do
-    it "Temporary Transaction Test" $ \ctx -> do
+    it "Check fixture transaction" $ \ctx -> do
         wid <- view walletId <$> fixtureWallet ctx
         r' <- request @ApiWallet ctx ("GET", "v2/wallets/" <> wid) Default Empty
-        let oneMillionAda = 1 * 1000 * 1000 * 1000 * 1000
         verify r'
             [ expectFieldEqual balanceTotal oneMillionAda
             , expectFieldEqual balanceAvailable oneMillionAda
             ]
+
+    it "Single Output Transaction" $ \ctx -> do
+        (wa, wb) <- (,) <$> fixtureWallet ctx <*> fixtureWallet ctx
```

Comment:
Why not write this over two lines?

Reply:
Hmmm.. Probably because conceptually, I still prefer using a monadic approach when the ordering of events is relevant. Here, using applicative makes it clear that these two actions could happen in any order, they don't depend on each other.

Reply:
Although I am fine with either ways, the do-notation is perhaps more readable.